cmd/list.go

```diff
@@ -13,11 +15,13 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var pkg_path = "./pkg/"
```

Comment:
Here i think it would better to use the template inside the binary like its made in the buider (see https://github.com/guervild/uru/blob/122d84b22173394f7be0609c453c2a52de66876c/pkg/builder/builder.go#L98).

Reply:
ya youre right, im not super familiar with go and its idiosyncrasy, but i remember seeing this in the code... good idea